### ISSUE TITLE
Iron

### ISSUE THREAD

**Issue by iteles** [dwyl/home#164]
We will need an iron with a reservatoire/tank for the most efficient ironing.

Here are a couple of options:

## Ferro com Caldeira TEFAL Fasteo SV6020E0 
https://www.worten.pt/pequenos-eletrodomesticos/tratamento-de-roupa/ferros-com-caldeira/ferro-com-caldeira-tefal-fasteo-sv6020e0-pressao-5-bar-jato-vapor-130-g-min-6488947
![image](https://user-images.githubusercontent.com/4185328/100520655-c252e500-3196-11eb-9eb6-17004b1cb70a.png)
![image](https://user-images.githubusercontent.com/4185328/100520661-ced73d80-3196-11eb-8c2c-855584be26a3.png)

The **major advantage** here is that it sits in its cradle which is a great safety feature and is a bit of a known quantity - there are 9 positive review (the most negative on says you knock the temperature gauge with your hand whilst ironing but I don't think this is insurmountable).

## Ferro com Caldeira UFESA PL2500 Space Compact
https://www.worten.pt/pequenos-eletrodomesticos/tratamento-de-roupa/ferros-com-caldeira/ferro-com-caldeira-ufesa-pl2500-space-compact-pressao-6-bar-jato-de-vapor-300-g-min-7278982
![image](https://user-images.githubusercontent.com/4185328/100520760-79e7f700-3197-11eb-8fff-22cedf92e244.png)
![image](https://user-images.githubusercontent.com/4185328/100520794-abf95900-3197-11eb-91df-48db1b3ad225.png)

**The major advantage:** Compact but still holds 1.9 litres which is considerably more than the TEFAL (which holds 1.2l)

**Disadvantage**: I can't find reviews for it anywhere and it doesn't even appear on the UFESA website (they're only showing the PL450 because it's quite new apparently) _and_ if anyone walks by, there is a potential for the hot portion of the iron to catch them and burn them. But this is easily mitigated by making sure ironing is done in the linen room only.

It's a shame because I _really_ like the compact design with the bigger water tank.

## FERRO CALDEIRA PHILIPS GC9330/20
Then there's a much higher end like this:
https://mediamarkt.pt/products/ferro-caldeira-philips-gc9330-20-7-bar
![image](https://user-images.githubusercontent.com/4185328/100524968-b249fe00-31b4-11eb-9081-24502eb49d53.png)
![image](https://user-images.githubusercontent.com/4185328/100525065-76fbff00-31b5-11eb-8b03-af4373b9d7ff.png)

But it's BIG and weighs 4.2kg _empty_. The boxed Tefal weighs 4kg so I would imagine it weighs a bit less itself.
Also, this just feels like overkill. 

These were the deals. There were others not on deals but didn't seem to offer anything above these.

I think we would we be better off getting the Tefal or the Ufesa for now and seeing if we _need_ to upgrade later on.

**Comment by nelsonic** [comment]
@iteles yeah, I can see the advantage of the more expensive one other than more water tank capacity. I would favour the Tefal for safety (less risk of burning by accidentally touching the iron in its upright position) even though it occupies more space. 💭

**Comment by iteles** [comment]
Purchased! To be delivered 👍

**Comment by iteles** [comment]
Got a message saying this had been dispatched :+1:
